docs/SUMMARY.md

```diff
@@ -0,0 +1,53 @@
+# Table of Contents
+
+* [CSI repository welcome](book_files/csi_block_storage_kc_welcome.md)
+* [What's new](book_files/csi_block_storage_kc_whatsnew.md)
+* [Release notes](book_files/csi_block_storage_kc_rn.md)
+	* [What's new in 1.6.0](content/release_notes/csi_rn_whatsnew.md)
+	* [Compatibility and requirements](content/release_notes/csi_rn_compatibility.md)
+		* [Supported storage systems](content/release_notes/csi_rn_supported_storage.md)
+		* [Supported operating systems](content/release_notes/csi_rn_supported_os.md)
+		* [Supported orchestration platforms](content/release_notes/csi_rn_supported_orchestration.md)
+	* [Change log](content/release_notes/csi_rn_changelog.md)
+		* [1.6.0 (June 2021)](content/release_notes/csi_rn_changelog_1.6.0.md)
+		* [1.5.0 (March 2021)](content/release_notes/csi_rn_changelog_1.5.0.md)
+		* [1.4.0 (December 2020)](content/release_notes/csi_rn_changelog_1.4.0.md)
+		* [1.3.0 (September 2020)](content/release_notes/csi_rn_changelog_1.3.0.md)
+		* [1.2.0 (June 2020)](content/release_notes/csi_rn_changelog_1.2.0.md)
+		* [1.1.0 (March 2020)](content/release_notes/csi_rn_changelog_1.1.0.md)
+		* [1.0.0 (November 2019)](content/release_notes/csi_rn_changelog_1.0.0.md)
+	* [Limitations](content/release_notes/csi_rn_limitations.md)
+	* [Known issues](content/release_notes/csi_rn_knownissues.md)
+* [Overview](content/csi_overview.md)
+* [Installation](content/installation/csi_ug_install_intro.md)
+	* [Compatibility and requirements](content/installation/csi_ug_requirements.md)
+	* [Installing the operator and driver](content/installation/csi_ug_install_operator.md)
+		* [Installing the driver with OpenShift web console](content/installation/csi_ug_install_operator_openshift.md)
+		* [Installing the driver with GitHub](content/installation/csi_ug_install_operator_github.md)
+		* [Installing the driver with OperatorHub.io](content/installation/csi_ug_install_operator_operatorhub.md)
+	* [Uninstalling the operator and driver](content/installation/csi_ug_uninstall.md)
+		* [Uninstalling the driver with the OpenShift web console](content/installation/csi_ug_uninstall_openshift.md)
```

Comment:
`Uninstalling the driver with the OpenShift web console` ->
`Uninstalling the driver with OpenShift web console`

Reply:
@rivkap could we at least make it consistent?
![image](https://user-images.githubusercontent.com/57443811/158076785-86daedf4-1323-4cd9-8bae-d089f17928fb.png)

Reply:
fixed in https://github.com/IBM/ibm-block-csi-driver/pull/466